plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkIndexByScrollRequestBuilder.java

```diff
@@ -0,0 +1,24 @@
+package org.elasticsearch.plugin.reindex;
```

Comment:
License header is missing; probably also for some more new files. Can you please check them?